.gitignore

```diff
@@ -16,6 +16,8 @@ venv
 ual_baus_install.sh
 configs/hlcm_renter_no_unplaced.yaml
 configs/hlcm_owner_no_unplaced.yaml
+configs/hlcm_renter_lowincome_no_unplaced.yaml
```

Comment:
@theocharides : these two files shouldn't be ignored since they should be a part of configs rather than large input files?

Reply:
Yeah it's confusing, basically rather than add these two files, ual.py creates them based on other yaml files. Fletcher added previous versions to gitignore. Clearly I have edited ual code, so I don't think this is necessary in order to avoid not touching ual code/files. But I think it helps the user because even if they edit these files, the model will try to write over them.